internal/common/environment.go

```diff
@@ -46,7 +46,7 @@ func GetNodeLabellerImages() NodeLabellerImages {
 	const kvmCpuNfdDefaultImage = "quay.io/kubevirt/cpu-nfd-plugin"
 
 	const libvirtDefaultVersion = "v0.21.0"
-	const libvirtDefaultImage = "kubevirt/virt-launcher"
+	const libvirtDefaultImage = "docker.io/kubevirt/virt-launcher"
```

Comment:
I would like to stay away from any hard bindings to `docker.io` given the recent pull limits and stuff

Reply:
Without the `docker.io` the image could not be pulled. Do you know any other way how I could fix it?

Reply:
I think there should be a plan to move this to `quay.io`

Reply:
We can change it, when the image in quay will be available. Currently I'm getting an error when I try to pull it form quay:
```
$ docker pull quay.io/kubevirt/virt-launcher:v0.21.0
Error response from daemon: unauthorized: access to the requested resource is not authorized
```

I'm not sure if the image does not exist, or if I don't have premissions.